app/main.py

```diff
@@ -1,6 +1,15 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    storage = {}
+
+    def wrapper(*args, **kwargs) -> Any:
+        if args in storage:
+            print("Getting from cache")
+            return storage.get(args)
```

Comment:
what if two different functions have the same `args` but different `kwargs`?
you must use `args` + `kwargs` as the keys of `storage `